tests/unit_tests/datasets/schema_tests.py

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=import-outside-toplevel, invalid-name, unused-argument, redefined-outer-name
+import pytest
+from marshmallow import ValidationError
+
+from superset.datasets.schemas import validate_python_date_format
+
+
+# pylint: disable=too-few-public-methods
+@pytest.mark.parametrize(
+    "payload",
+    [
+        None,
+        "epoch_ms",
+        "epoch_s",
+        "%Y/%m/%dT%H:%M:%S.%f",
```

Comment:
@mapledan could you move this test case to `test_validate_python_date_format_raises`.